js/viz.js

```diff
@@ -325,13 +351,11 @@ const viz = {
 
   dragEnd() {
     d3.event.subject.shadow = false;
-    this.endEvent();
```

Comment:
I think this solves some of the issues however on update the whole graph is still recalculated, so perhaps this isn't the right change?